### PR TITLE
Feature/update map ui

### PR DESCRIPTION
**Pull request by GiftLanga**
created map toggle buttons

### CODE REVIEW

File: waterlog/src/Components/Map.js [hunk 1]
```diff
@@ -69,56 +71,81 @@ class MapComponent extends Component {
...
+        <div className="map-icon-button-div-layer2">
+          <div className="map-button-div-layer2 map-button-tab">
+            <button
+              className={`map-button ${this.state.simpleView ? "" : "active"}`}
+              onClick={() => {
+                this.setState({ simpleView: false });
+              }}
+            >
+              Simplified
           </button>
```
Comment (FredJEhlers) [inline waterlog/src/Components/Map.js]: Incorrect indentation. align with the opening button tag

File: waterlog/src/Components/Map.js [hunk 2]
```diff
@@ -69,56 +71,81 @@ class MapComponent extends Component {
...
-          <button
-            className={`map-button ${this.state.simpleView ? "active" : ""}`}
-            onClick={() => {
-              this.setState({ simpleView: true });
-            }}
-          >
-            Live Map
+            <button
```
Comment (FredJEhlers) [inline waterlog/src/Components/Map.js:117]: The closing button tag is misaligned

File: waterlog/src/Stylesheets/_map.scss [hunk 3]
```diff
@@ -21,20 +20,43 @@
...
+  max-width: 700px;
+  position: absolute;
+}
+
 .map-button-div-layer2 {
-	z-index: 2;
-	bottom: 0;
+  z-index: 2;
+  bottom: 0%;
+	margin: 9px auto;
```
Comment (FredJEhlers) [inline waterlog/src/Stylesheets/_map.scss:33]: indentation

File: waterlog/src/Stylesheets/_map.scss [hunk 4]
```diff
@@ -21,20 +20,43 @@
...
+  position: absolute;
+  width: 100%;
+}
+
+.map-icon-div-layer2-fullscreen {
 	width: 100%;
```
Comment (FredJEhlers) [inline waterlog/src/Stylesheets/_map.scss:39]: indentation

File: waterlog/src/Stylesheets/_map.scss [hunk 5]
```diff
@@ -21,20 +20,43 @@
...
+  right: 0;
 }
 
+.invisible {
+  display: none;
+}
 .map-button-tab {
-	overflow: hidden;
+  overflow: hidden;
 	button {
```
Comment (FredJEhlers) [inline waterlog/src/Stylesheets/_map.scss:57]: Space between declarations and indentation

Comment (FredJEhlers) [reply waterlog/src/Stylesheets/_map.scss:57]: Just fix all indentations in the this file

Comment (FredJEhlers) [reply waterlog/src/Stylesheets/_map.scss:57]: empty line EOF

**Review by FredJEhlers — APPROVED** [review]

File: waterlog/src/Stylesheets/_map.scss (hunk 4 above)
Comment (GiftLanga) [reply waterlog/src/Stylesheets/_map.scss:39]: indentation fixed

File: waterlog/src/Stylesheets/_map.scss (hunk 5 above)
Comment (GiftLanga) [reply waterlog/src/Stylesheets/_map.scss:57]: indentation fixed

File: waterlog/src/Stylesheets/_map.scss (hunk 3 above)
Comment (GiftLanga) [reply waterlog/src/Stylesheets/_map.scss:33]: indentation fixed

File: waterlog/src/Components/Map.js (hunk 2 above)
Comment (GiftLanga) [reply waterlog/src/Components/Map.js:117]: indentation fixed

File: waterlog/src/Components/Map.js (hunk 1 above)
Comment (GiftLanga) [reply waterlog/src/Components/Map.js]: indentation fixed

**Pull request merged**